includes/sanitizers/class-amp-img-sanitizer.php

```diff
@@ -157,30 +157,29 @@ private function determine_dimensions( $need_dimensions ) {
 				if ( ! $node instanceof DOMElement ) {
 					continue;
 				}
+				$height = $dimensions && isset( $dimensions['height'] ) ? $dimensions['height'] : self::FALLBACK_HEIGHT;
+				$width  = $dimensions && isset( $dimensions['width'] ) ? $dimensions['width'] : self::FALLBACK_WIDTH;
```

Comment:
I think  `$this->args['content_max_width']` should be used here if available. It got removed in #979.